README.md

```diff
@@ -1 +1,131 @@
+```
+Mac/Linux:
+```bash
+chmod +x ./infra/scripts/*.sh 
+```
+Windows:
+```Powershell
+set-executionpolicy remotesigned
+```
+Run the follow command to provision resoruces in Azure
+```bash
+azd provision
+```
+
+If you don't run azd provision, you can create an [OpenAI resource](https://portal.azure.com/#create/Microsoft.CognitiveServicesOpenAI) and an [AI Search resource](https://portal.azure.com/#create/Microsoft.Search) in the Azure portal to get your endpoints. After it deploys, click Go to resource and view the Endpoint value.  You will also need to deploy a model, e.g. with name `chat` with model `gpt-35-turbo` and `embeddings` with model `text-embedding-3-small`
+
+### Create local.settings.json (Should be in the same folder as host.json. Automatically created if you ran azd provision)
+```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet-isolated",
+    "AZURE_OPENAI_ENDPOINT": "<paste from above>",
+    "CHAT_MODEL_DEPLOYMENT_NAME": "chat",
+    "AZURE_AISEARCH_ENDPOINT": "<paste from above>",
+    "EMBEDDING_MODEL_DEPLOYMENT_NAME": "embeddings",
+    "SYSTEM_PROMPT": "You must only use the provided documents to answer the question"
+    }
+}
+```
+
+### Permissions
+#### Add your account (contoso.microsoft.com) with the following permissions to the Azure OpenAI and AI Search resources when testing locally.
+If you used `azd provision` this step is already done - your logged in user and your function's managed idenitty already have permissions granted. 
+* Cognitive Services OpenAI User (OpenAI resource)
+* Azure Search Service Contributor (AI Search resource)
+* Azure Search Index Data Contributor (AI Search resource)
+ 
+
+### Access to Azure OpenAI and Azure AI Search with virtual network integration
+If you selected virtual network integration, access to Azure OpenAI and Azure AI Search is limited to the Azure Function app through private endpoints and cannot be reached from the internet. To allow testing from your local machine, you need to go to the networking tab in Azure OpenAI and Azure AI Search and add your client ip to the allowed list. If you used `azd provision` this step is already done.
+
+## Run your app using Visual Studio Code
+
+1. Open the folder in a new terminal.
+1. Run the `code .` code command to open the project in Visual Studio Code.
+1. In the command palette (F1), type `Azurite: Start`, which enables debugging without warnings.
+1. Press **Run/Debug (F5)** to run in the debugger. Select **Debug anyway** if prompted about local emulator not running.
+1. Send POST requests to the `ingest` and `ask` endpoints respectively using your HTTP test tool. If you have the [RestClient](https://marketplace.visualstudio.com/items?itemName=humao.rest-client) extension installed, you can execute requests directly from the [`test.http`](./app/test.http) project file.
+
+## Run your app using Visual Studio
```

Comment:
Please cut entire Visual Studio section.  It's not really a strong client for python

Reply:
fixed